{"frames":[{"character_number":0,"end_character_number":0,"end_line_number":0,"file":"","func":"getPTC(DB_source_data, 0)","line_number":0},{"character_number":9,"end_character_number":122,"end_line_number":17,"file":"<text>","func":"read.table(paste(folder, target, sep = \"\"), sep = \"|\", header = TRUE, \n    stringsAsFactors = FALSE, row.names = NULL)","line_number":17},{"character_number":0,"end_character_number":0,"end_line_number":0,"file":"","func":"file(file, \"rt\")","line_number":0}],"message":"Error in file(file, \"rt\") : cannot open the connection\n"}